### ISSUE TITLE
Add support for stroke gradients, e.g. for line series

### ISSUE THREAD

**Issue by tsvetomir** [telerik/kendo-ui-core#2013]
### Current behavior
- The chart provides no means to set a gradient for line series color
- The Drawing API doesn't support specifying a gradient as [stroke color](http://docs.telerik.com/kendo-ui/api/javascript/drawing/stroke-options)

### Desired behavior
- Gradient instances should be supported for line series
- Setting gradients should be allowed for Drawing element stroke

### Reason for Suggesting the Feature
We already support gradients for fills. The infrastructure is in place to enable it for stroke as well.

**Comment by danielkaradachki** [comment]
Will not be complex to implement but there will be inconsistency with the fill configuration. The stroke has mandatory width option so we cannot expect the gradient to be passed for the stroke option. We should expect it for the stroke.color.

To be honest I am not sure why we expect the gradient to be set directly to the fill option. Fill opacity works along with fill gradient. Perhaps we should allow both and set some default width for stroke gradients.

**Comment by tsvetomir** [comment]
Agreed, the gradient should go to `stroke.color` and remain independent from `width` and `opacity`.

You're right about the fill opacity, SVG allows you to set `fill="url(#Gradient1)" fill-opacity="0.5"` so it technically allows both. We can add an `opacity` option on the gradients to allow this scenario.
